test/engine/update.result

```diff
@@ -850,3 +850,285 @@ s:update({1}, {{'=', 'field3', {d = 30, e = 31, f = 32}}})
 s:drop()
 ---
 ...
+--
+-- gh-3378: allow update absent nullable fields
+-- '!'
+s = box.schema.create_space('test', {engine = engine})
```

Comment:
You don't really need spaces, indexes, or any other storage things. The updates work on tuples. So for non-formatted tuples you could simply use `t = box.tuple.new(...)` and `t:update(...)`.

Reply:
Yes, but this is is engine test, and @Korablev77 write (see his comment above) that i need to check both memtx and vinyl engines. So i need to create space with specified engine.

Reply:
This is an engine test only because you made so. You could have a separate file, or use box/ folder. Talking of the "need to check both engines" - why? It is not related to engines anyhow as I demonstrated. But up to you.